7_Classes/singleton_class.py

```diff
@@ -0,0 +1,23 @@
+""" object:
+    The base class of the class hierarchy.
+
+    When called, it accepts no arguments and returns a new featureless
+    instance that has no instance attributes and cannot be given any.
+    """
+
+
+class SingletonClass(object):
```

Comment:
looks like you copied this example from some old resources :)
there's no need to inherit classes from `object` python 3.

ps: I found this example in geeks for geeks under py3 title. but still, there's no need to inherit from `object`.

Reply:
it's better to create a singleton class as an abstract class and add it as a metaclass to the required.
https://refactoring.guru/design-patterns/singleton/python/example